dependency_support/org_theopenroadproject/org_theopenroadproject.bzl

```diff
@@ -21,15 +21,9 @@ def org_theopenroadproject():
     maybe(
         new_git_repository,
         name = "org_theopenroadproject",
-        commit = "21b84678f4eb0eae2e38b897e4e0d36e145f99cd",
+        commit = "65053db0233ffb6e3235b3f28774afbcd063d569",
```

Comment:
Can we merge past commit https://github.com/The-OpenROAD-Project/OpenROAD/commit/6eab677861f92ec4699ddc2233c7b57ef701778e ?